cellpack/autopack/Environment.py

```diff
@@ -1158,21 +1018,45 @@ def setCompartmentMesh(self, compartment, ref_obj):
                 filename=ref_obj, vertices=vertices, faces=faces, vnormals=vnormals
             )
 
-    def create_ingredient(self, **arguments):
-        ingredient_type = arguments["Type"]
-        if ingredient_type == "SingleSphere":
-            ingr = SingleSphereIngr(**arguments)
-        elif ingredient_type == "MultiSphere":
+    def create_ingredient(self, recipe, **arguments):
+        ingredient_type = arguments["type"]
+
+        if ingredient_type == "single_sphere":
+            radius = arguments["radius"]
+            arguments = IOutils.IOingredientTool.clean_arguments(
```

Comment:
this won't be needed once we have the migration code working, because things will either be in the new format, or converted to the new format